modules/test/playwright/tests/style-book-web/stylebookEditor.spec.ts

```diff
@@ -233,3 +220,242 @@ test('LPD-35561 Preview StyleBook when edit StyleBook', async ({
 		});
 	});
 });
+
+test(
+	'Preview style book on pages',
+	{tag: '@LPD-35560'},
+	async ({
+		page,
+		pageEditorPage,
+		pagesAdminPage,
+		productMenuPage,
+		site,
+		styleBooksPage,
+	}) => {
+		const firstPageName = getRandomString();
+		const secondPageName = getRandomString();
+		const thirdPageName = getRandomString();
+
+		const styleBookName = getRandomString();
+
+		const previewIframe = page.frameLocator(
+			'iframe.style-book-editor__page-preview-frame'
+		);
+
+		await test.step('Add the first page with a Blogs widget', async () => {
+			await productMenuPage.goToPages();
+
+			await pagesAdminPage.createNewPage({
+				addButtonLabel: 'Page',
+				draft: true,
+				name: firstPageName,
+				template: 'Blank',
+			});
+
+			await pageEditorPage.addWidget('Collaboration', 'Blogs');
+		});
+
+		await test.step('Add the second page with a My Sites widget', async () => {
+			await productMenuPage.goToPages();
+
+			await pagesAdminPage.createNewPage({
+				addButtonLabel: 'Page',
+				draft: true,
+				name: secondPageName,
+				template: 'Blank',
+			});
+
+			await pageEditorPage.addWidget('Community', 'My Sites');
+		});
+
+		await test.step('Add a paragraph fragment to the third page and publish', async () => {
+			await productMenuPage.goToPages();
+
+			await pagesAdminPage.createNewPage({
+				addButtonLabel: 'Page',
+				draft: true,
+				name: thirdPageName,
+				template: 'Blank',
+			});
+
+			await pageEditorPage.addFragment('Basic Components', 'Paragraph');
+
+			await pageEditorPage.publishPage();
+		});
+
+		await test.step('Add a heading fragment to the first page and publish', async () => {
+			await page.getByRole('link', {name: firstPageName}).click();
+
+			await pageEditorPage.addFragment('Basic Components', 'Heading');
+
+			await pageEditorPage.publishPage();
+		});
+
+		await test.step('Add a button fragment to the second page and publish', async () => {
+			await page.getByRole('link', {name: secondPageName}).click();
+
+			await pageEditorPage.addFragment('Basic Components', 'Button');
+
+			await pageEditorPage.publishPage();
+		});
```

Comment:
Why aren't these two steps done when creating the first and second page?

Reply:
@thiago-buarqque I'm not sure. The original poshi test does it this way.

Reply:
Looking at it more, the only thing that makes sense is they wanted to test if drafts were saved.